src/app/features/reminders/reminder-card/reminder-card.component.ts

```diff
@@ -0,0 +1,67 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import Status, { StatusDesc } from '../../../core/enums/status.enum';
+import { Reminder } from '../../../core/models/reminder.model';
+
+@Component({
+  selector: 'app-reminder-card',
+  templateUrl: './reminder-card.component.html',
+  styleUrls: ['./reminder-card.component.scss']
+})
+export class ReminderCardComponent implements OnInit {
+  @Input() reminder!: Reminder;
+  @Input() save!: () => void;
+  @Input() reminders: Reminder[];
+  @Output() remindersUpdate = new EventEmitter<Reminder[]>();
+
+  constructor() {
+    this.reminders = [];
+  }
+
+  ngOnInit(): void {
+    this.reminders = this.getReminders();
+  }
+
+  remove(id: number) {
+    let reminder = this.reminders?.find(r => r.id == id)!;
+    if (reminder) {
+      reminder.deleted = true;
+      this.update();
+      this.save();
```

Comment:
Here you're using the JS Reference, but considering a good practice you may avoid it when possible. For this case, you should send the reminder object to the update method.

Reply:
Alright, got it